src/gvars.c

```diff
@@ -825,6 +881,34 @@ static Obj FuncIsReadOnlyGVar (
     return IsReadOnlyGVar(GVarName(CSTR_STRING(name))) ? True : False;
 }
 
+/****************************************************************************
+**
+*F  IsConstantGVar( <gvar> ) . . . . . . return if a variable is a constant
+*/
+Int IsConstantGVar(UInt gvar)
+{
+    return INT_INTOBJ(ELM_GVAR_LIST(WriteGVars, gvar)) == -1;
```

Comment:
Better (and slightly safer...) code can be generated if you instead write `return ELM_GVAR_LIST(WriteGVars, gvar) == INTOBJ_INT(-1);` -- this would then also match `IsReadOnlyGVar` above.